app/views/shared/_outage_alert.html.erb

```diff
@@ -0,0 +1,9 @@
+<div class="margin-bottom-2">
```

Comment:
added the margin classes but the spacing is different from what it looked like previously